app/main/spawn.js

```diff
@@ -26,7 +26,29 @@ const CONTENT_SERVICE_PATH = path.join(SERVICES_PATH, 'content-service', 'index.
 // @TODO Will `node` be accessible via path on users machines that don't
 // have node installed as an engineer?
 
-const children = [];
+const children = new Set();
+function spawnProcess(name, command, args, options) {
```

Comment:
Should maybe make this shared, possibly augmenting the existing spawn function we have in build/utils?